torch/csrc/Generator.cpp

```diff
@@ -95,12 +95,12 @@ static PyObject * THPGenerator_setState(THPGenerator *self, PyObject *_new_state
   using namespace torch::autograd;
   HANDLE_TH_ERRORS
   if (!THPVariable_Check(_new_state)) {
-    throw TypeError("expected a torch.ByteTensor, but got %s", Py_TYPE(_new_state)->tp_name);
+    throw torch::TypeError("expected a torch.ByteTensor, but got %s", Py_TYPE(_new_state)->tp_name);
```

Comment:
Don't you want to use `TORCH_CHECK_TYPE` here?

Reply:
The formatting between the `torch::TypeError()` and `c10::TypeError()` is done very differently. Since I wasn't sure of the impact, I only changed the ones I know were problematic.
Do you think I should change these as well?

Reply:
I think in general it would be better if we try to avoid doing printf style formatting as much as possible, since there is no error checking for when we get it wrong.

Reply:
Yes, let me open an issue for that.